modules/ROOT/pages/hardware-and-software-requirements.adoc

```diff
@@ -9,20 +9,29 @@ If you plan to install Mule and run it on premises, review these minimum hardwar
 == Minimum Hardware Requirements
 
 * 2 GHz CPU or 1 virtual CPU in virtualized environments
-* 1 GB of RAM. Add RAM to match your latency requirements, or to support large simultaneous messages or a large number of simultaneous messages if your applications process them.
+* 1 GB of RAM
 * 4 GB of storage
 
-== Supported Software and Systems
+Adjust RAM to match your latency requirements and the size and number of simultaneous messages that applications process.
 
-Verify that you use a supported version of Java and a supported operating system before you install Mule.
+== Required Software
 
-Mule Runtime runs as a standalone server; therefore an application server is not required. However, if you use an application server or need to access a data store, verify that you use a supported application server or database.
+Verify that you use a supported version of Java and a supported operating system before you install Mule.
 
 [%header,cols="15a,85a"]
 |===
 |Software |Version
 | JDK | JDK 1.8.0 (1.8.0_202 recommended)
 | OS | MacOS 10.11.x, HP-UX 11i V3, AIX 7.2, Windows 2016 Server (recommended), Windows 2012 R2 Server, Windows 10 (recommended), Windows 8.1, Solaris 11.3, RHEL 7, Ubuntu Server 18.04, Linux Kernel 3.13+
+|===
+
+== Supported Software
+
+Mule Runtime runs as a standalone server; therefore an application server is not required. However, if you use an application server or need to access a data store, verify that you use a supported application server or database.
```

Comment:
Suggestion. I usually avoid semi-colons and stick to periods and commas in cases like this. I suspect that people who speak English as a second language are not as accustomed to semi-colons. I believe this was something I learned in a tech writing class but don't have a reference for that info... So this is just a suggestion.  

"... standalone server, so an application .." 

"... standalone server. Therefore, an..."

or if you use "; therefore" add a comma after "therefore" (see https://onlinewritingtraining.com.au/2018/07/25/however-therefore-furthermore/ and https://grammarist.com/grammar/therefore/):

" standalone server; therefore ..."